getting-started/singleframe.rst

```diff
@@ -111,7 +111,10 @@ Running single frame processing
 
    As mentioned in :doc:`part 1 <data-setup>`, this part of the processing is by far the most time consuming.
    If you do not wish to process all the data in the repository at this time, you can specify a data query that will reduce the number of exposures to be processed.
-   Simply add the argument ``-d "instrument='HSC' AND detector=41 AND exposure=322"`` to the command line below.
+   Simply add the argument ``-d "instrument='HSC' AND detector=41 AND exposure=322"`` to the command line below, and change ``pipelines/DRP.yaml#singleFrame`` to ``pipelines/DRP.yaml#simpleSingleFrame``.
+
+   Note that this will give you an idea of how to execute processing steps from the command line, but **this subset of data will not be sufficient for the full tutorial. If you wish to follow the entire tutorial, you will need to use the full** ``rc2_subset`` **dataset.**
```

Comment:
I don't know if the bold text is really necessary here.

Also, I think we generally prefer "one sentence per line" to make diffs simpler:

https://developer.lsst.io/restructuredtext/style.html#text-wrapping

Reply:
Yeah, maybe boldface was a bit much. Removed it, but if users continue to have the same issues, we may want to make the warning stronger.

Fixed the "one sentence per line" issue.